### DISCUSSION TITLE
GHES AND Email notification template

### DISCUSSION THREAD

**Discussion by timansky** [community/community#51093] [Enterprise]
### Select Topic Area

Product Feedback

### Body

Hi.
There is no way to change templates for email notification?
For example contacts/server name/logo (header/footer)

Also images are not working if server is private (not available from internet), generated urls is something like this:
`https://ci4.googleusercontent.com/proxy/hZNMA1GDot-VtFAU8uuVjQXu809SBp1NfxiZftnIabhXVdUpty8xgtxFDe6ndbH1NF81wNR23Y93otpOE3VN6JvwSa-cMD2ZS4LUL5WXPmAQYbqkSUo=s0-d-e1-ft#https://github.domain.local/images/email/global/octocat-logo.png`

URL is forced to get image from google proxy.

<img width="619" alt="Screenshot 2023-03-03 at 19 36 31" src="https://user-images.githubusercontent.com/1217278/227873161-31691c23-04fb-4795-8afb-687d9467ca3e.png">

GHES version 3.8.0

**Comment by queenofcorgis** [comment]
Thanks for the feedback @timansky! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by akimsir** [comment]
Are there any ways to solve the problem with images in emails here?
